defaults/initrd.scripts

```diff
@@ -1553,6 +1571,71 @@ splash() {
 	return 0
 }
 
+splashCmd() {
+	case "${1}" in
+		init)
+		if [ "${FBSPLASH}" = '1' ]
+		then
+			splash 'init'
+		elif [ "${PLYMOUTH}" = '1' ]
+		then
+			plymouthInit
+		fi
+		;;
+
+		verbose)
+		if [ "${FBSPLASH}" = '1' ]
+		then
+			splash 'verbose'
+		elif [ "${PLYMOUTH}" = '1' ]
+		then
+			plymouth_running && run plymouth --hide-splash
+		fi
+		;;
+
+		set_msg)
+		if [ "${FBSPLASH}" = '1' ]
+		then
+			splash 'set_msg' "${2}"
+		elif [ "${PLYMOUTH}" = '1' ]
+		then
+			plymouth_running && run plymouth --update="${2}"
+		fi
+		;;
+
+		quiet)
+		plymouth_running && run plymouth --show-splash
+		;;
+
+		hasroot)
+		plymouth_running && run plymouth --newroot="${2}"
+		;;
+	esac
+}
+
+plymouth_running() {
```

Comment:
```
plymouthRun() {
	run plymouth --ping 2>/dev/null || return $?
	eval run plymouth "${@}" 2>/dev/null
}
```

Reply:
I really like this; in next commit.  Also resolves your previous comment.